### PR TITLE
Adressing is_accessible problem in roh_mhmm, issue #296

### PR DESCRIPTION
**Pull request by hardingnj**
Supersedes #300 

Thanks for your work on this @olavurmortensen. Apologies for the delay in review.

- Merged in subsequent changes to master
- Fixed for consistency of this change with poisson function
- minor spelling changes
- slight edits for clarity in tests

### CODE REVIEW

**Comment by alimanfoo** [comment]
Also @hardingnj would be great if you could add an entry in the release notes (docs/release.rst) crediting yourself and @olavurmortensen.

**Comment by hardingnj** [comment]
All should be done! Always happy to relearn how to `git rebase` properly :)

**Comment by olavurmortensen** [comment]
Cool, nice to see this done :)

File: docs/release.rst [hunk 1]
```diff
@@ -22,6 +22,10 @@ WIP v1.3.0
 * Minor documentation fixes. By :user:`Murillo Rodrigues <mufernando>`, 
   :issue:`273`, :issue:`294`.
 
+* Improve handling of `None` when defining accessibility in `roh_mhmm` 
+  and `roh_poissonhmm`. By :user:`Ólavur Mortensen <olavurmortensen>` 
+  and :user:`Nick Harding <hardingnj>`, issue `296`.
```
Comment (alimanfoo) [inline docs/release.rst:27]:
```suggestion
  and :user:`Nick Harding <hardingnj>`, :issue:`296`.
```

**Comment by alimanfoo** [comment]
Thanks @olavurmortensen and @hardingnj :-)

**Pull request merged**